spec/order_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'order'
+
+describe Order do
```

Comment:
To be even clearer, you could separate out the tests with more describe blocks.
`describe '#add_to_order' do` 
Would just make it easier to read and maybe if you came back to it, you could find a troubleshoot all the tests for one method easier.